README.md

```diff
@@ -41,6 +40,26 @@ pen & paper RPG game design".
 * Docker
 
 
+## User Interface
+
+The webpage contains short project description and two forms with instructions.
+
+![empty-forms](./readme_images/empty-forms.png)
+
+In the manual-input form, every field has question mark sign next to it. Hovering over it displays a prompt showing the acceptable range of values.
+
+![filled-forms](./readme_images/filled-forms.png)
+
+After submission, the calculated level value is displayed. Below appears a new form for modifying the level value.
+
+![results](./readme_images/results.png)
+
+A table with 5 possible modifications appears. Modified features in every set are marked with green bold font. 
+In addition, the user can load new features to the form above or download it in JSON format.
```

Comment:
download them to a JSON file (wiem, że w dokumentacji mamy wszędzie "plik w formacie JSON", ale wg mnie "plik JSON" jest bardziej po ludzku i i tak wiadomo o co chodzi, ale możesz chyba też nie zmieniać tej części - grunt żeby było "them" zamiast "it")